src/protocol/protocol.ts

```diff
@@ -697,7 +697,9 @@ export namespace Script {
   };
 
   export type AddPreloadScriptResult = {
-    script: PreloadScript;
+    result: {
+      script: PreloadScript;
+    };
```

Comment:
```
script.AddPreloadScriptResult = {
  script: script.PreloadScript
}
```

```
script.PreloadScript = text;
```

`result` is not in the spec

Reply:
{
  id: js-uint,
  result: script.AddPreloadScriptResult,
}

Reply:
It looks like the return types need some refactoring, as they are quite confusing.

Reply:
Agreed. I can do so in a follow-up CL.